galaxy/api/serializers/users.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from collections import OrderedDict
+
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+
+from rest_framework import serializers
+
+from .serializers import BaseSerializer
+
+logger = logging.getLogger(__name__)
+
+User = get_user_model()
+
+USER_FIELDS = (
+    'id', 'url', 'related', 'summary_fields', 'created', 'modified',
+    'username', 'staff', 'full_name', 'date_joined', 'avatar_url',
+    'notify_survey', 'notify_import_fail', 'notify_import_success',
+    'notify_content_release', 'notify_author_release',
+    'notify_galaxy_announce',
+)
+
+
+class MeSerializer(BaseSerializer):
```

Comment:
Class name could be better (e.g. `CurrentUserSerializer` or `ActiveUserSerializer`)